lib/blockscore/person.rb

```diff
@@ -10,7 +10,7 @@ class Person < Base
 
     attr_reader :question_sets
 
-    def initialize(options = {})
+    def initialize(*)
```

Comment:
Actually, that's a good point. In this case, `*` and `_` are functionally equivalent due to `super`.